pykafka/simpleconsumer.py

```diff
@@ -461,6 +525,19 @@ def _handle_success(parts):
                     [part for errcode, parts in parts_by_error.iteritems()
```

Comment:
Given that `partitions` has been dropped from the code above this, I think L523-525 may need rewriting to write into `owned_partition_offsets` instead?